CSS/README.md

```diff
@@ -162,18 +162,22 @@
 
 - [x] 28. Describe pseudo-elements and discuss what they are used for.
 
-  > It's to style a part of an element, like `::first-letter` or `::before`. They can be used to add a special symbol before a paragraph, change color of first character of a line, etc.
+  > - It's to style a part of an element, like `::first-letter` or `::before`. They can be used to add a special symbol before a paragraph, change color of first character of a line, etc.
+  > - These are appropriately called pseudo "elements" (not selectors) because they don't select any "real" element that exists on the page
 
 - [x] 29. What's HTML5 shim, what's its usage
 
   > - The HTML5 Shiv enables use of HTML5 sectioning elements in legacy Internet Explorer and provides basic HTML5 styling for Internet Explorer 6-9, Safari 4.x (and iPhone 3.x), and Firefox 3.x.
   > - Pretty much not needed now as most of the browser as on _12 Sep 2017_ supports sectioning elements
 
 - [x] 30. What is Media Queries and why it's used?
-  
-  > Media queries are useful when you want to apply CSS styles depending on a device's general type (such as print vs. screen), specific characteristics (such as the width of the browser viewport). With the huge variety of internet-connected devices available today, media queries are a vital tool for building websites and apps that are robust enough to work on whatever hardware your users have.
 
-  > Targeting sample code below
+  > - The goal of responsive design is creating the right experience for the right device.
+  > - With a gazillion different devices on the market, this can be a big task. Media queries helps us achieve our goal of creating right experience for the right device. 
+  > - CSS Media queries allow you to target CSS rules based on - for instance - screen size, device-orientation or display-density. This means you can use CSS Media Queries to tweak a CSS for an iPad, printer or create a responsive site.
```

Comment:
`You` can be written as third person view like `us`. Just a suggestion.

Reply:
addressed